core/templates/dev/head/pages/exploration_editor/ExplorationStatesService.js

```diff
@@ -23,15 +23,13 @@ oppia.factory('ExplorationStatesService', [
   'ExplorationInitStateNameService', 'AlertsService', 'ChangeListService',
   'StateEditorService', 'ValidatorsService', 'StatesObjectFactory',
   'SolutionValidityService', 'AngularNameService',
-  'AnswerClassificationService', 'ContextService',
-  'UrlInterpolationService',
+  'AnswerClassificationService', 'ContextService', 'UrlInterpolationService',
   function(
       $log, $uibModal, $filter, $location, $rootScope, $injector, $q,
       ExplorationInitStateNameService, AlertsService, ChangeListService,
       StateEditorService, ValidatorsService, StatesObjectFactory,
-      SolutionValidityService, AngularNameService,
-      AnswerClassificationService, ContextService,
-      UrlInterpolationService) {
+      SolutionValidityService, AngularNameService, AnswerClassificationService,
+      ContextService, UrlInterpolationService) {
```

Comment:
Line breaks in deps here should match those above.

Reply:
Done!